Looks like this breaks everything but the Windows builds.
Linux:
```
net.cpp: In member function 'bool CConnman::InitBinds(const std::vector<CService>&, const std::vector<CService>&)':
net.cpp:2257:25: error: variable 'CConnman::InitBinds(const std::vector<CService>&, const std::vector<CService>&)::in_addr6 inaddr6_any' has initializer but incomplete type
         struct in_addr6 inaddr6_any = IN6ADDR_ANY_INIT;
                         ^~~~~~~~~~~
make[2]: *** [libbitcoin_server_a-net.o] Error 1
Makefile:5954: recipe for target 'libbitcoin_server_a-net.o' failed
```
macOS:
```
libbitcoin_server_a-net.o `test -f 'net.cpp' || echo './'`net.cpp
net.cpp:2257:25: error: variable has incomplete type 'struct in_addr6'
        struct in_addr6 inaddr6_any = IN6ADDR_ANY_INIT;
                        ^
net.cpp:2257:16: note: forward declaration of 'in_addr6'
        struct in_addr6 inaddr6_any = IN6ADDR_ANY_INIT;
               ^
1 error generated.
make[2]: *** [libbitcoin_server_a-net.o] Error 1
Makefile:5954: recipe for target 'libbitcoin_server_a-net.o' failed
```

--- a/src/net.cpp
+++ b/src/net.cpp
@@ -2254,7 +2254,8 @@ bool CConnman::InitBinds(const std::vector<CService>& binds, const std::vector<C
     if (binds.empty() && whiteBinds.empty()) {
         struct in_addr inaddr_any;
         inaddr_any.s_addr = INADDR_ANY;
-        fBound |= Bind(CService((in6_addr)IN6ADDR_ANY_INIT, GetListenPort()), BF_NONE);
+        struct in6_addr inaddr6_any = IN6ADDR_ANY_INIT;
+        fBound |= Bind(CService(inaddr6_any, GetListenPort()), BF_NONE);
         fBound |= Bind(CService(inaddr_any, GetListenPort()), !fBound ? BF_REPORT_ERROR : BF_NONE);
     }
     return fBound;

